src/components/text-editor/__internal__/toolbar/toolbar.component.tsx

```diff
@@ -80,24 +126,22 @@ const Toolbar = ({
   );
 
   useEffect(() => {
-    if (focusIndex === null) {
-      setTabbable(true);
-    }
-  }, [focusIndex]);
+    isFirstRender.current = false;
```

Comment:
I'll copy this here because it applies to this comment too. 

`Awh it's left over from when I was debugging the broken keyboard nav with the toolbar. It was doing some crazy stuff with setting the focusIndex to null which was giving me bugs. I'll delete it.`